proto/types.proto

```diff
@@ -11,3 +11,15 @@ message ExtraData {
 	bytes JsonSpec = 1;
 	google.protobuf.Any RuncOptions = 2;
 }
+
+// Message to specify network config for a Firecracker VM
+message FirecrackerNetworkConfig {
+	string MAC = 1;
+	string HostDevName = 2;
+}
+
+// Message to specify the various Firecracker VM configurations for a task
+message FirecrackerConfig {
+	google.protobuf.Any RuncOptions = 1; // Can be used by the CreateTask request needs to relay any Runc options to the agent
+	FirecrackerNetworkConfig NetworkConfig = 2; // Can be used to set the networking configuration for a VM
```

Comment:
We might want this to be `repeated` so that more than one device can be configured.

Reply:
Yeah, I had it that way initially. Somehow missed that here. I'll modify it to handle multiple interfaces. Good catch!